### PR TITLE
Seedlet: Fix IE11 issues

### PR DESCRIPTION
**Pull request by MaggieCabrera**
#### Changes proposed in this Pull Request:

This PR addresses two errors that appear on IE11 for Seedlet:
- Fixes the invalid character error by removing the backticks
- Adds crossorigin="anonymous" to all CSS tags in an attempt to fix the CORS issue that currently is only affecting production

#### Related issue(s):

#3043

### CODE REVIEW

**Comment by MaggieCabrera** [comment]
D55506-code -> diff

File: seedlet/functions.php [hunk 1]
```diff
@@ -483,3 +483,11 @@ function seedlet_author_bio() {
  * Load Jetpack compatibility file.
  */
 require get_template_directory() . '/inc/jetpack.php';
+
+/**
+ * Adds crossorigin anonymous to all css scripts so that the IE11 ponyfill doesn't have CORS issues
+ */
+function seedlet_css_crossorigin( $html, $handle ) {
+    return str_replace( "media='all'", "media='all' crossorigin='anonymous'", $html );
```
Comment (scruffian) [inline seedlet/functions.php]: Should we only do this in IE?

**Review by scruffian — APPROVED** [review]
LGTM. I guess we have to test on production...

File: seedlet/functions.php (hunk 1 above)
Comment (MaggieCabrera) [reply seedlet/functions.php]: yes, that makes sense

**Comment by scruffian** [comment]
🚢

**Comment by MaggieCabrera** [comment]
I removed the crossorigin function since that doesn't work but left the backtick fix for IE. This was already on prod, merging this so the reverted function gets to .com with the next batch of fixes for seedlet

**Pull request merged**